### DISCUSSION TITLE
How can I increase the amount of log messages and warnings (unstable branch)?

### DISCUSSION THREAD

**Discussion by thowe-switzerland** [rtl-airband/RTLSDR-Airband#252] [Q&A] [answered]
When I download and build version 3.2.1 as tar.gz, I get quite a lot of log output and warnings when I configure something questionable. When I check out and build the head of unstable, I get almost no log output.

I have looked around in the documentation and source code and have not figured out on the first go how I could increase the number of log messages again.

So: How can I increase the amount of log messages and warnings (unstable branch)?

Thanks for any hints on this.

**Comment by szpajder** [comment, marked as answer]
Almost nothing has changed since 3.2.1 as far as logging verbosity is concerned.

Can you give an example of messages that you see with 3.2.1 and you expect to see with unstable, but you don't?

**Reply by thowe-switzerland** [reply]
I now wanted to demonstrate the difference and rebuilt unstable (4.0.0pre) again according to the readme. And indeed I see no difference now. This time I see similar log outputs for both versions. Almost certainly not like that before.  

Thus: Done for now. Sorry and thank you!
